saleor/graphql/product/bulk_mutations/products.py

```diff
@@ -168,22 +168,10 @@ def clean_variant_input(
 
         cost_price_amount = cleaned_input.pop("cost_price", None)
         if cost_price_amount is not None:
-            if cost_price_amount < 0:
```

Comment:
Should we add `validate_price_amount` here?

Reply:
Yes, thanks

Reply:
Done